css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+    </ul>
+
+    <ol style="display: contents;" data-expectedrole="list" data-testname="ol with display:contents has list role (child li has display:contents)" class="ex">
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li as child of ol, both with display:contents, has listitem role" class="ex">x</li>
+        <li>y</li>
+    </ol>
+
+    <ol style="display: contents;" data-expectedrole="list" data-testname="ol with display:contents has list role" class="ex">
+        <li>x</li>
+        <li data-expectedrole="listitem" data-testname="li, as child of ol with display:contents, has listitem role" class="ex">y</li>
+    </ol>
+
+    <ol>
+        <li>x</li>
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li with display:contents, as child of ol, has listitem role" class="ex">y</li>
+    </ol>
+
+    <div role="list" style="display: contents;" data-expectedrole="list" data-testname="div with list role and display:contents has list role" class="ex">
+        <div role="listitem" data-expectedrole="listitem" data-testname="div with listitem role, as child of div with display:contents, has listitem role" class="ex">x</div>
+        <div>y</div>
+    </div>
+
+    <div role="list" style="display: contents;" data-expectedrole="list" data-testname="div with list role and display:contents has list role (child div with listitem role has display:contents)" class="ex">
+        <div>x</div>
+        <div role="listitem" style="display: contents;" data-expectedrole="listitem" data-testname="div with listitem role (as child of div with list role), both with display:contents, has listitem role" class="ex">y</div>
+    </div>
+
+    <div role="list">
+        <div>x</div>
+        <div role="listitem" style="display: contents;" data-expectedrole="listitem" data-testname="div with listitem role with display:contents, as child of div with list role, has listitem role" class="ex">y</div>
+    </div>
+
+    <!-- Lists: description lists -->
+    <dl>
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <dl style="display: contents;">
+        <dt style="display: contents;" data-expectedrole="term" data-testname="dt with display:contents, as child of dl with display:contents, has term role" class="ex">q</dt>
+        <dd style="display: contents;" data-expectedrole="definition" data-testname="dd with display:contents, as child of dl with display:contents, has definition role" class="ex">r</dd>
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
```

Comment:
These description list tests all fail in all browsers for reasons unrelated to `display:contents`, and test case failures ([`html-aam/roles.html`](https://wpt.fyi/results/html-aam/roles.html?label=master&label=experimental&aligned&q=label%3Aaccessibility) and possibly more) are already tracking these. Let's cut them from this particular PR. Add a commented link/todo if you prefer.

Reply:
Done: included in this latest commit: fb97fe9db208360ff58b96d4b70af2be24a1e6f4.